From a469204e7e30fca50642d0626646d0aa3ee73873 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Wed, 15 Mar 2023 11:30:28 +0200
Subject: [PATCH] [SW-136144] ami: fw_loader: include specs_h9 directory

The ami host hl-fw-loader does not include the CMakeList
file present in hl-fw-loader application repo hence it
needs to include specs_h9 directory for the compilation.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Id5214e916494aaf2e62f852f2a7ca37e23d225aa
---

diff --git a/package/hl-fw-loader/hl-fw-loader.mk b/package/hl-fw-loader/hl-fw-loader.mk
index e0b647e..4b7cceb 100644
--- a/package/hl-fw-loader/hl-fw-loader.mk
+++ b/package/hl-fw-loader/hl-fw-loader.mk
@@ -3,7 +3,7 @@
 
 HL_FW_LOADER_DEPENDENCIES = libopenssl pciutils zlib eeprom-util dtc hlmi lz4
 
-HL_FW_LOADER_INC = $(SPECS_ROOT) -I $(SPECS_EMBEDDED_ROOT) \
+HL_FW_LOADER_INC = $(SPECS_ROOT) -I $(SPECS_EMBEDDED_ROOT) -I $(SPECS_ROOT_H9) \
 	  -I ${EMBEDDED_ROOT} -I ${EMBEDDED_ROOT}/hlmi/include
 
 LZ4_SRC_PATH = ${ZEPHYR_MODULES_ROOT}/lz4/lib
